bson/src/main/org/bson/RawBsonDocument.java

```diff
@@ -52,8 +52,19 @@
     private static final long serialVersionUID = 1L;
     private static final int MIN_BSON_DOCUMENT_SIZE = 5;
 
+    /**
+     * The raw bytes.
+     */
     private final byte[] bytes;
+
+    /**
+     * The offset into bytes, which much be less than {@code bytes.length}.
+     */
     private final int offset;
+
+    /**
+     * The length, which much be less than {@code offset + bytes.length}.
```

Comment:
much -> must. The same is applicable to the doc comment for the `offset` field above.

Reply:
Done